javascript-timers/timeout.js

```diff
@@ -0,0 +1,4 @@
+
+setTimeout(() => {
+  document.getElementsByTagName("h1")[0].innerHTML = "Hello There"
```

Comment:
Directly setting `innerHTML` is dangerous and slow. Please use `textContent` instead.